docs/support-guide.md

```diff
@@ -0,0 +1,243 @@
+# OpenShift Provider Certification Tool - Support Guide
+
+> WIP: this document is working in progress
+
+- [Support Case Check List](#check-list)
+  - [New Support Cases](#check-list-new-case)
+  - [New Executions](#check-list-new-executions)
+- [Setting up the Review Environment](#setup)
+  - [Install tools](#setup-install)
+  - [Download dependencies](#setup-download-dependencies)
+  - [Download Partner Results](#setup-download-results)
+- [Review guide: exploring the failed tests](#review-process)
+  - [Exploring the failures](#review-process-exploring)
+  - [Extracting the failures to local directory](#review-process-extracting)
+  - [Explaning the extracted files](#review-process-explain)
+  - [Review Guidelines](#review-process-guidelines)
+
+
+## Support Case Check List <a name="release"></a>
+
+### New Support Cases <a name="release"></a>
+
+Check-list to require when **new** support case has been opened:
+
+- Documentation: Installing Steps containing the flavors/size of the Infrastructure and the steps to install OCP
+- Documentation: Diagram of the Architecture including zonal deployment
+- Archive with Certification results
+- Archie with must-gather
+- Installation Check List (file `use-installation-checklist.md`) with the details of the installation
+
+### New Executions <a name="release"></a>
+
+The following assets, certification assets, should be updated when certain conditions happen:
+
+- Certification Results
+- Must Gather
+- Install Documentation (when any item/flavor/configuration has been modified)
+
+
+The following conditions require new certification assets:
+
+- The version of OpenShift Container Platform has been updated
+- Any Infrastructure component(s) (e.g.: server size, disk category, ELB type/size/config) or cluster dependencies (e.g.: external storage backend for image registry) have been modified
+
+
+## Review Environment <a name="release"></a>
+
+### Setting up local environment <a name="release"></a>
+
+> TODO: link with each tool URL
+
+- Download the OPCT
+- Download the omg
+
+### Download dependencies <a name="release"></a>
+
+- Download the Baseline execution for the version used by the partner
+
+> TODO: Link to GDrive path
+
+- Download the suite test list for the version used by the partner
+
+> TODO: Currently the test list is not included in the resulting archive, need to create the steps to extract the openshift-tests util from the same release used by the partner, and extract the test list.
+
+### Download Partner Results <a name="release"></a>
+
+- Download the Provider certification archive
+- Download the Must-gather
+
+## Review guide: exploring the failed tests <a name="release"></a>
+
+> NOTE/Question: It's not clear if we could keep that section on the `user-results-review.md` instead of one dedicated to supporting - as it can be used in the future by partners and support.
```

Comment:
I think we should keep it in this document for now. In my opinion, if we intend a guide to be used by partners and support, they would have basically the same steps.

Reply:
Ok, indeed. Agreed! The comment was removed and the section intro was added.